symbols/overlay29.yml

```diff
@@ -691,6 +770,22 @@ overlay29:
         r0: entity pointer
         r1: move category index (0 for physical, 1 for special)
         return: boost
+    - name: TickNoSlipCap
+      address:
+        NA: 0x230FB90
+        EU: 0x2310604
+      description: |-
+        Checks if the entity is a team member and holds the No-Slip Cap, and if so attempts to make one item in the bag sticky.
+        
+        r0: pointer to entity
+    - name: AdvanceStatusAndHealthRegen
```

Comment:
Can we call this `TickStatusAndHealthRegen` to be consistent with `TickNoSlipCap`?